_data/errors.yml

```diff
@@ -118,4 +118,80 @@ oidc:
         We are no longer supporting the id_token_hint attribute as a method of logging out.
         ##### What to do:
         - Update your logout request to utilize the client_id attribute.
-        More information about the logout request parameters can be found here: [https://developers.login.gov/oidc/#logout-request](https://developers.login.gov/oidc/#logout-request){:target="_blank"}.
+        More information about the logout request parameters can be found here: [https://developers.login.gov/oidc/logout/#logout-request](https://developers.login.gov/oidc/logout/#logout-request){:target="_blank"}.
+  token:
+    - title: Code is expired
+      id: token-expired
+      accordion-id: token_expired
+      content: |
+        ##### Why it's happening
+        In the authentication response, a `code` value is returned that is then used in the `token` endpoint request. "The code is intended to be used within 15 minutes immediately after it is generated. This error may mean that the user paused for too long during the authentication attempt.
+        ##### What to do:
+        - Try another authentication attempt without pausing.
+    - title: Invalid audience claim
+      id: invalid-aud-claim
+      accordion-id: invalid_aud_claim
+      content: |
+        ##### Why it's happening
+        The `aud` value in the `client_assertion` JWT is not correct.
+        ##### What to do:
+        - Check the `aud` key in the `client_assertion` JWT that is being sent in your `token` request.
+        Integration environment value: https://idp.int.identitysandbox.gov/api/openid_connect/token
```

Comment:
Maybe we can wrap these this and below environment values in the backtick code formatting?

Reply:
updated in https://github.com/18F/identity-dev-docs/pull/393/commits/92cab9801b2f1663abd39187d50d81f3ca0f1320